src/js/container.js

```diff
@@ -1,4 +1,5 @@
-import { Container } from "inversify";
+import "reflect-metadata"; // Needed for inversify
```

Comment:
Just out of curiosity: why wasn't that needed before?

Reply:
We didn't use `extension-kid` it has been just sitting there. With this PR we do import the library thus engaging `@injectable`-like decorators.